src/components/FullWindowOverlay.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { PropsWithChildren, ReactNode } from 'react';
+import { Platform, StyleProp, View, ViewStyle } from 'react-native';
+
+// Native components
+import FullWindowOverlayNativeComponent from '../fabric/FullWindowOverlayNativeComponent';
+const NativeFullWindowOverlay: React.ComponentType<
+  PropsWithChildren<{
+    style: StyleProp<ViewStyle>;
+  }>
+> = FullWindowOverlayNativeComponent as any;
+
+function FullWindowOverlay(props: { children: ReactNode }) {
+  if (Platform.OS !== 'ios') {
+    console.warn('Importing FullWindowOverlay is only valid on iOS devices.');
```

Comment:
Maybe we should change it to `Using FullWindowOverlay ...`. But we can do it in another PR ofc

Reply:
Sure thing, I'll do it here since this is a pretty small change 👍

Reply:
Done in https://github.com/software-mansion/react-native-screens/pull/1982/commits/2a6eedd1c027a51f328bab3034d3e5a3988cc6a8